reactDashboard/src/components/SingleValue/SingleValueBar.css

```diff
@@ -59,31 +82,67 @@
     flex-direction: row;
     flex-wrap: wrap;
   }
+  .region-single-value-container .single-value-heading {
```

Comment:
Why the difference in dimensions for the regional single values ?

Reply:
Because they are not in the same size container on each page and we need separate styling to stop text overlapping for responsive views - I couldn't get what worked for one to work for the other

Reply:
I've added a comment in the style sheets to explain

Reply:
refactored css to split the specifics for overview and regional pages